epochStart/metachain/validators.go

```diff
@@ -474,6 +480,13 @@ func (vic *validatorInfoCreator) saveValidatorInfo(miniBlock *block.MiniBlock) {
 		if err != nil {
 			log.Debug("validatorInfoCreator.saveValidatorInfo.Put", "hash", validatorInfoHash, "error", err)
 		}
+
+		err = vic.epochStartStaticStorage.Put(validatorInfoHash, marshalledData)
+		if err != nil {
+			log.Debug("validatorInfoCreator.epochStartStaticStorage.Put", "hash", validatorInfoHash, "error", err)
+		}
+
+		log.Trace("saveValidatorInfo: put validatorInfo into epochStartStaticStorage", "validatorInfo hash", validatorInfoHash)
```

Comment:
this will be printed even though one or both `Put` operations failed
not sure if intended or not